contentcuration/contentcuration/view/internal_views.py

```diff
@@ -238,21 +238,27 @@ def convert_data_to_nodes(content_data, parent_node):
     try:
         root_mapping = {}
         sort_order = 1
```

Comment:
If there may already be nodes in there, shouldn't this start at the current `max_sort_order` + 1, rather than 1?